independent-projects/bootstrap/classloader-commons/src/main/java/io/quarkus/commons/classloading/ClassLoaderHelper.java

```diff
@@ -18,4 +22,7 @@ public static String fromClassNameToResourceName(final String className) {
         return className.replace('.', '/').concat(".class");
     }
 
+    public static boolean isInJdkPackage(String name) {
+        return name.startsWith(JAVA) || name.startsWith(JDK_INTERNAL) || name.startsWith(SUN_MISC);
+    }
```

Comment:
I would assume this is correct, but since I am not 100% (not have I seen other CL does this), I would prefer the additional packages being in a separate commit so we can revert if we find some obscure case where this change fails

Reply:
I sure can do that.

Reply:
🙏🏼

Reply:
I moved it to another commit and rebased.